packages/auth/.env.example

```diff
@@ -1,4 +1,4 @@
-DB_DISABLE_SSL=
+DB_ENABLE_SSL=
```

Comment:
Not sure if we should include this optional field in the examples or just skip it, I chose to include it

Reply:
Yeah I think it's good to include for completeness. Please add a note that the release process needs to include some manual changes to environment variables (the process is somewhat arduous)

Reply:
I will add it (by the way I am the release manager this week). Could you explain the process to me in person (or point me to related documentation)?

Reply:
For sure. This is the best documentation so far, but I will take you through it in person and we can add a doc just for that process

Reply:
https://docs.beyondessential.com.au/books/dev-ops/page/tupaia-main-server-structure-overview#bkmrk-environment-variable